pkg/controller/common/util.go

```diff
@@ -114,24 +111,13 @@ func SetAnnotation(resource metav1.Object, annotation, value string) {
 var initOperatorNamespace sync.Once
 var operatorNamespace string
 
-// XXX: remove after updating to newer version of operator-sdk
-// from newer version of operator-sdk
+// GetOperatorNamespace initializes and caches this operator's namespace; panics on error
 func GetOperatorNamespace() string {
 	initOperatorNamespace.Do(func() {
-		if ns := os.Getenv("POD_NAMESPACE"); ns != "" {
```

Comment:
Maybe we should leave this in; we added this to run the operator locally as if it was deployed to `POD_NAMESPACE` without having to write to `/var/run/secrets/kubernetes.io/serviceaccount/namespace` locally.

Reply:
Added it back in, if `ErrNoNamespace` is returned.